src/style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars > * {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
```

Comment:
It is better to set `margin-right: 4px` or `margin-left: 4px`. We usually try to set margins for one side, this way it is easier to maintain code in one style

Reply:
changed to margin-right